guides/doc-Upgrading_and_Updating/topics/updating_satellite_server_to_next_minor_version.adoc

```diff
@@ -73,103 +73,195 @@ If you lose connection to the command shell where the upgrade command is running
 
-. Ensure that the `rubygem-foreman_maintain` package that provides `{foreman-maintain}` is installed and up to date:
+Perform the following steps on the disconnected {ProjectServer}
+
+. Copy the generated `{Project}-repos.tgz` file to your disconnected {ProjectServer}
+. Extract the archive to anywhere accessible by the `root` user.
+In the following example `/root` is the extraction location.
 +
-[options="nowrap"]
+[options="nowrap" subs="attributes"]
 ----
-# yum install rubygem-foreman_maintain
+# cd /root
+# tar zxf {Project}-repos.tgz
 ----
+. Create a Yum configuration file under `/etc/yum.repos.d` with the following repository information:
++
+[options="nowrap" subs="attributes"]
+----
+[{RepoRHEL7ServerAnsible}]
+name=Ansible {SatelliteAnsibleVersion} RPMs for Red Hat Enterprise Linux 7 Server x86_64
+baseurl=file:///root/{Project}-repos/{RepoRHEL7ServerAnsible}
+enabled=1
+
+[{RepoRHEL7Server}]
+name=Red Hat Enterprise Linux 7 Server RPMs x86_64
+baseurl=file:///root/{Project}-repos/{RepoRHEL7Server}
+enabled=1
+
+[{RepoRHEL7ServerSatelliteServerProductVersion}]
+name={ProjectNameX} for RHEL 7 Server RPMs x86_64
+baseurl=file:///root/{Project}-repos/{RepoRHEL7ServerSatelliteServerProductVersion}
+enabled=1
+
+[{RepoRHEL7ServerSatelliteMaintenanceProductVersion}]
+name={ProjectName} Maintenance 6 for RHEL 7 Server RPMs x86_64
+baseurl=file:///root/{Project}-repos/{RepoRHEL7ServerSatelliteMaintenanceProductVersion}
+enabled=1
 
+[{RepoRHEL7ServerSoftwareCollections}]
+name=Red Hat Software Collections RPMs for Red Hat Enterprise Linux 7 Server x86_64
+baseurl=file:///root/{Project}-repos/{RepoRHEL7ServerSoftwareCollections}
+enabled=1
+----
++
+. In the configuration file, replace the `/root/{Project}-repos` with the extracted location.
 . Check the available versions to confirm the next minor version is listed:
 +
-[options="nowrap" subs="+quotes,attributes"]
+[options="nowrap" subs="attributes"]
 ----
 # {foreman-maintain} upgrade list-versions
 ----
-
-. Use the health check option to determine if the system is ready for the upgrade.
-On the first use of this command, `{foreman-maintain}` prompts you to enter the hammer admin user credentials and saves them in the `/etc/foreman-maintain/foreman-maintain-hammer.yml` file.
++
+. Use the health check option to determine if the system is ready for upgrade.
+On first use of this command, `{foreman-maintain}` prompts you to enter the hammer admin user credentials and saves them in the `/etc/foreman-maintain/foreman-maintain-hammer.yml` file.
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {foreman-maintain} upgrade check --target-version {ProjectVersion}.__z__
+# {foreman-maintain} upgrade check --whitelist="check-upstream-repository,repositories-validate" --target-version {ProductVersion}.__z__
 ----
 +
 Review the results and address any highlighted error conditions before performing the upgrade.
 
```

Comment:
```suggestion
+
```
And below. 
Can you preview locally and ensure this all flows as one procedure?

Reply:
Addressed this comment by making it a `.` instead of `+` to be consistent with what was  on line 45.
Check out 
https://partha.fedorapeople.org/docs/build/Upgrading_and_Updating/index-satellite.html#_updating_disconnected_satelliteserver 
and let me know.